tests/prague/eip7685_general_purpose_el_requests/test_deposits_withdrawals_consolidations.py

```diff
@@ -5,10 +5,11 @@
 """  # noqa: E501
 
 from itertools import permutations
-from typing import Any, Dict, Generator, List, Tuple
+from typing import Dict, Generator, List, Tuple
```

Comment:
I think we should take the opportunity to rename this file to something more generic like `test_multi_type_requests.py`, so it can be updated in the future if a new request type is added.